configure: error: C++ compiler cannot create executables

18:10:51 ┌ tesseract git ............................ [Recently updated]
18:10:51 Installing docbook-xsl... done
18:11:05 Installing libarchive... done
18:11:07 ├ Running autogen...
18:11:33 ├ Running configure...
Likely error:
CFLAGS: -mthreads -mtune=generic -O2 -pipe
LDFLAGS: -pipe -static-libgcc -static-libstdc++
../configure --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --prefix=/local64 --disable-shared --enable-static --bindir=/local64/bin-global --disable-graphics --disable-tessdata-prefix LIBLEPT_HEADERSDIR=/local64/include LIBS=-LC:/MABS/local64/lib -LC:/MABS/msys64/mingw64/lib -LC:/MABS/local64/lib -LC:/MABS/msys64/mingw64/lib -liconv -llept -lz -lpng16 -lz -ltiff -ljpeg -llzma -lz -lzstd --datadir=/local64/bin-global
configure: loading site script /mingw64/etc/config.site
checking for x86_64-w64-mingw32-g++... x86_64-w64-mingw32-g++
checking whether the C++ compiler works... no
configure: error: in `/build/tesseract-git/build-64bit':
configure: error: C++ compiler cannot create executables
See `config.log' for more details
configure failed. Check C:/MABS/build/tesseract-git/build-64bit/ab-suite.configure.log
This is required for other packages, so this script will exit.
18:11:35   Creating diagnostics file...

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -396,6 +396,7 @@ unset _deps
 
 _check=(libtiff{.a,-4.pc})
 if enabled_any libwebp libtesseract &&
+    do_pacman_install libjpeg-turbo
     do_vcs "https://gitlab.com/libtiff/libtiff.git"; then
     do_uninstall "${_check[@]}"
     sed -i 's/Libs.private.*/& -ljpeg -llzma -lz -lzstd/' libtiff-4.pc.in
@@ -544,7 +545,6 @@ fi
 _check=(libFLAC{,++}.{,l}a flac{,++}.pc)
 [[ $standalone = y ]] && _check+=(bin-audio/flac.exe)
 if [[ $flac = y ]] && do_vcs "https://git.xiph.org/flac.git"; then
-    # release = #tag=1.3.1
     do_pacman_install libogg
     do_autogen
     if [[ $standalone = y ]]; then

